cli/src/pcluster/config/update_policy.py

```diff
@@ -188,6 +188,23 @@ def is_managed_placement_group_deletion(change, patch):
     ) and not is_placement_group_managed_for_compute_resource(target_q_networking, target_cr_networking)
 
 
+def get_managed_fsx_from_config(config):
+    """Extract managed Fsx for Lustre shared storage entries in a cluster configuration."""
+    managed_fsx_storage = [
+        storage
+        for storage in config.get("SharedStorage", [])
+        if storage.get("StorageType") == "FsxLustre" and "FileSystemId" not in storage.get("FsxLustreSettings", {})
+    ]
+    return managed_fsx_storage
+
+
+def unchanged_managed_fsx_lustre_names(_, patch):
+    """Get list of managed Fsx for Lustre Shared Storage that hasn't changed between cluster configuration updates."""
+    fsx_names_before_update = {fsx.get("Name") for fsx in get_managed_fsx_from_config(patch.base_config)}
```

Comment:
nit - what about `managed_fsx_names_before_update , so to highlight that only contains managed fsx names?